Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/inbound/deferred/request/StandardInboundDocSubmissionDeferredRequest.java

```diff
@@ -88,36 +90,31 @@ public StandardInboundDocSubmissionDeferredRequest(
 
     @Override
     @InboundProcessingEvent(beforeBuilder = DocSubmissionBaseEventDescriptionBuilder.class,
-            afterReturningBuilder = DocSubmissionArgTransformerBuilder.class,
-            serviceType = "Document Submission Deferred Request",
-            version = "")
+        afterReturningBuilder = DocSubmissionArgTransformerBuilder.class,
+        serviceType = "Document Submission Deferred Request", version = "")
     public XDRAcknowledgementType provideAndRegisterDocumentSetBRequest(ProvideAndRegisterDocumentSetRequestType body,
-            AssertionType assertion) {
-
-        auditRequestFromNhin(body, assertion);
+        AssertionType assertion, Properties webContextProperties) {
 
         XDRAcknowledgementType response = processDocSubmissionRequest(body, assertion);
 
-        auditResponseToNhin(response, assertion);
+        auditResponse(body, response, assertion, webContextProperties);
 
         return response;
     }
 
     @Override
     XDRAcknowledgementType processDocSubmissionRequest(ProvideAndRegisterDocumentSetRequestType body,
-            AssertionType assertion) {
+        AssertionType assertion) {
         XDRAcknowledgementType response = null;
 
-        auditRequestToAdapter(body, assertion);
-
         String localHCID = getLocalHCID();
         if (isPolicyValid(body, assertion, localHCID)) {
-            try{
-            	LOG.debug("Policy Check Succeeded");
-            	getDocSubmissionUtils().convertDataToFileLocationIfEnabled(body);
-            	response = sendToAdapter(body, assertion);
-            } catch(LargePayloadException lpe){
-            	LOG.error("Failed to retrieve payload document.", lpe);
+            try {
+                LOG.debug("Policy Check Succeeded");
+                getDocSubmissionUtils().convertDataToFileLocationIfEnabled(body);
+                response = sendToAdapter(body, assertion);
+            } catch (LargePayloadException lpe) {
+                LOG.error("Failed to retrieve payload document.", lpe);
```

Comment:
Since you've already modified this file, would you mind adjusting to match our standard exception logging format?

LOG.error("msg: " + lpe.getLocalizedMessage(), lpe);